src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java

```diff
@@ -0,0 +1,93 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+/**
+ * A class to be the primary key of a {@link Ticket} for JPA, which requires
+ * every Entity to have a single primary key.
+ *
+ * Declared to implement Serializable because "Composite-id class must implement
+ * Serializable."
+ *
+ * @author Al-amine AHMED MOUSSA
+ */
+
+@Embeddable
+public class TicketIdentity implements Serializable {
+	/*
+	 * Serialization version. Increment on any change to class structure that is
+	 * pushed to production.
+	 */
+	private static final long serialVersionUID = 1L;
+	/*
+	 * the flight ID.
+	 */
+	private Integer flight;
+	/*
+	 * the row of the seat .
+	 */
+	@Column(name = "seat_row")
+	private Integer row;
+	/*
+	 * the seat group.
+	 */
+	private Character seat;
+
+	/*
+	 * the Getters and Setters of the Class
+	 */
+	public Integer getFlight() {
+		return flight;
+	}
+
+	public Integer getRow() {
+		return row;
+	}
+
+	public Character getSeat() {
+		return seat;
+	}
+
+	/*
+	 * the Constructors of the Class we need empty Constructor for Hibernate Jpa
+	 * Purposes
+	 */
+
+	public TicketIdentity() {
+
+	}
+
+	public TicketIdentity(Integer flight, Integer row, Character seat) {
+		this.flight = flight;
+		this.row = row;
+		this.seat = seat;
+	}
+
+	@Override
+	public int hashCode() {
+		return Objects.hash(flight, row, seat);
+	}
+
+	@Override
+	public boolean equals(Object obj) {
+		if (this == obj)

```

Comment:
Use braces around `if` and `else` blocks here as well (and in my opinion refactoring to the same pattern as you now use in `Ticket::equals` would be an improvement as well, but that's a less important issue).